core.go

```diff
@@ -101,7 +101,7 @@ func (c Core) NewMultipartUpload(bucket, object string, opts PutObjectOptions) (
 
 // ListMultipartUploads - List incomplete uploads.
 func (c Core) ListMultipartUploads(bucket, prefix, keyMarker, uploadIDMarker, delimiter string, maxUploads int) (result ListMultipartUploadsResult, err error) {
-	return c.listMultipartUploadsQuery(bucket, keyMarker, uploadIDMarker, prefix, delimiter, maxUploads)
+	return c.listMultipartUploadsQuery(context.Background(), bucket, keyMarker, uploadIDMarker, prefix, delimiter, maxUploads)
```

Comment:
suggest adding a WithContext variant, for this and other calls in core.go that do not pass context currently.